application/account-management/WebApp/shared/translations/locale/nl-NL.po

```diff
@@ -0,0 +1,281 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2024-10-07 23:40+0200\n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: nl-NL\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: \n"
+"X-Generator: @lingui/cli\n"
+
+msgid "Account"
+msgstr "Account"
+
+msgid "Account settings"
+msgstr "Accountinstellingen"
+
+msgid "Account Settings"
+msgstr "Accountinstellingen"
+
+msgid "Actions"
+msgstr "Acties"
+
+msgid "Active Users"
+msgstr "Actieve gebruikers"
+
+msgid "Active users in the past 30 days"
+msgstr "Actieve gebruikers in de afgelopen 30 dagen"
+
+msgid "Add more in the Users menu"
+msgstr "Meer toevoegen via het gebruikersmenu"
+
+msgid "Added"
+msgstr "Toegevoegd"
+
+msgid "All Users"
+msgstr "Alle gebruikers"
+
+msgid "An error occurred while processing your request. {0}"
+msgstr "Er is een fout opgetreden bij het verwerken van uw verzoek. {0}"
+
+msgid "By continuing, you accept our policies"
+msgstr "Door verder te gaan, accepteer je onze voorwaarden"
+
+msgid "Can't find your code? Check your spam folder."
+msgstr "Kun je je code niet vinden? Controleer je spammap."
+
+msgid "Cancel"
+msgstr "Annuleren"
+
+msgid "Continue"
+msgstr "Doorgaan"
```

Comment:
This shbould either be `Volgende` or `Verder`

Reply:
I prefer `Volgende`

Reply:
Just saw you also have `Next`, which is `Volgende`. So this one could probably be `Verder`.